lib/include/tick/array/abstractarray1d2d.h

```diff
@@ -14,14 +14,32 @@
 #include <algorithm>
 #include <atomic>
 #include <cstring>
+#include <memory>
 #include <type_traits>
 #include <typeinfo>
 
+#include "alloc.h"
 #include "promote.h"
 #include "vector_operations.h"
 
+// clang-format off
+// Don't touch this!
```

Comment:
Why is this necessary already? Cereal headers raise warnings? Maybe it worths noting it here

Reply:
It was necessary before I added "SortIncludes: False" to the clang format rules, re-arranging includes is not recommended sometimes

Reply:
still needed it seems,